mock/mock.go

```diff
@@ -18,6 +18,9 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+// regex for GCCGO functions
+var gccgoRE = regexp.MustCompile("\\.pN\\d+_")
```

Comment:
Use backquote to reduce escaping.

```suggestion
var gccgoRE = regexp.MustCompile(`\.pN\d+_`)
```